packages/assertions/src/crypto.ts

```diff
@@ -0,0 +1,7 @@
+import { SOLANA_ERROR__CRYPTO__RANDOM_VALUES_FUNCTION_UNIMPLEMENTED, SolanaError } from '@solana/errors';
+
+export async function assertPRNGIsAvailable() {
```

Comment:
Hmm… I wonder why some of these were made `async`. Seems like a mistake we shouldn't replicate here.